firestore/integration_test.go

```diff
@@ -1743,6 +1743,72 @@ func TestIntegration_FieldTransforms_Set(t *testing.T) {
 
 type imap map[string]interface{}
 
+func TestIntegration_Serialize_Deserialize_WatchQuery(t *testing.T) {
+	shouldBeFoundID := collectionIDs.New()
+
+	// Create documents for partiontioned query result
+	ctx := context.Background()
+	h := testHelper{t}
+	client := integrationClient(t)
+	cr1 := client.Collection(shouldBeFoundID)
+	dr1 := cr1.Doc("should-be-found-1")
+	h.mustCreate(dr1, map[string]string{"some-key": "should-be-found"})
+	defer h.mustDelete(dr1)
+
+	dr1.Collection(shouldBeFoundID)
+	dr2 := cr1.Doc("should-be-found-2")
+	h.mustCreate(dr2, map[string]string{"some-key": "should-be-found"})
+	defer h.mustDelete(dr2)
+
+	partitionedQueries, err := client.CollectionGroup(shouldBeFoundID).GetPartitionedQueries(ctx, 10)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	qProtoBytes, err := partitionedQueries[0].Serialize()
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Can blocks like this be pulled into a function?

Reply:
Done